packages/typespec-vscode/src/telemetry/telemetry-client.ts

```diff
@@ -43,8 +43,14 @@ export class TelemetryClient {
       );
       this._client = undefined;
     } else {
-      // has to convert the TelemetryReporter to any, otherwise it will report error: This expression is not constructable.
-      this._client = new (TelemetryReporter as any)(key);
+      try {
+        this._client = new TelemetryReporter.default(key);
```

Comment:
this is anoying this library is very much garbage, we already import as default. Does it however now not have the type issue?

Reply:
No. the library remain unchanged. so we have to do this.